harvard-lts/fits
# LIBDRS-9048 Update to OTS 1.0.62

**Resolves Issue**: [LIBDRS-9050](https://jira.huit.harvard.edu/browse/LIBDRS-9050)

# What does this Pull Request do?
Updates the OTS dependency to 1.0.62.
Removes the JDom 1 exclusion from the POM since OTS has been upgraded to JDom2.

# How should this be tested?
Using a Docker image to run the tests see the [README.md](https://github.com/harvard-lts/fits/blob/main/README.md)
```
docker build -f docker/Dockerfile-test -t fits-test .
docker run --rm -v `pwd`:/fits -v ~/.m2:/root/.m2 fits-test mvn clean test

```

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? n/a
- integration tests? n/a

# Interested parties
@cvicary @awoods @pwinckles
